wagtail/admin/templates/wagtailadmin/shared/ajax_pagination_nav.html

```diff
@@ -1,7 +1,7 @@
 {% load i18n %}
 {% load wagtailadmin_tags %}
 
-<div class="pagination">
+<nav class="pagination" aria-label="pagination">
```

Comment:
This needs to be

```suggestion
<nav class="pagination" aria-label="{% trans 'Pagination' %}">
```

so the content is translatable.